### PR TITLE
Tweak documentation to prepare for pausing community contributions

### PR DESCRIPTION
**Pull request by kytrinyx**
This updates the README and tweaks the issue template to document changes in the process.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -15,26 +30,22 @@ If you find a repository for your language, check the `active` key's value in th
...
-If new maintainers have questions, it will often be because these files are confusing or missing information.
+We are available to guide you along the way, and you can also get help from the community of folks who have done this for other languages.
 
-When that happens we should tweak the documentation for clarity in a new pull request to the request-new-language-track repository (not their repository) and tag the maintainer to review it.
-If it is still confusing, they will know, and figuring out how to explain it will help us fix it.
+## It doesn't exist - I want to create it!
 
-In some cases, we might discover that we're missing high-level documentation that should live in the [docs][docs] repo, in which case we should open an issue or pull request there, proposing the new documentation.
-Tag the new maintainer there, as well, to get their input on the new docs.
+Awesome!! Please create a new post in the [Exercism Forum][forum-support] so we can discuss the details.
```
Comment (iHiD) [inline README.md:47]:
```suggestion
Awesome! Please create a new post in the [Exercism Forum][forum-support] so we can discuss the details.
```

**Comment by ErikSchierboom** [comment]
Just to be clear, this repo is replacing the `request-new-language-track-repo`? If so, I'll need to update https://github.com/exercism/org-wide-files/pull/276/files

**Comment by kytrinyx** [comment]
> Just to be clear, this repo is replacing the request-new-language-track-repo? If so, I'll need to update https://github.com/exercism/org-wide-files/pull/276/files

Ah, excellent catch. Yes, I renamed the repository.

**Comment by ErikSchierboom** [comment]
I've updated the org-wide-files PR

**Pull request merged**